packages/components/psammead-section-divider/src/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import styled from 'styled-components';
+import { oneOf, shape, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN, // 600px
+  GEL_GROUP_5_SCREEN_WIDTH_MIN, // 1280px
+  MEDIA_QUERY_TYPOGRAPHY,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING, // 8px
+  GEL_SPACING_DBL, // 16px
+  GEL_SPACING_TRPL, // 24px
+} from '@bbc/gel-foundations/spacings';
+import {
+  getDoublePica,
+  GEL_FF_REITH_SANS,
+} from '@bbc/gel-foundations/typography';
+import {
+  C_CLOUD_LIGHT, // #BABABA
+  C_EBON, // #222222
+  C_WHITE, // #FFFFFF
+} from '@bbc/psammead-styles/colours';
+
+const MARGIN_TOP_PX = 1;
+const MARGIN_TOP_REM_DANGLING = 1.25;
+
+const halfLineHeightRem = group => (group.lineHeight / 2 + MARGIN_TOP_PX) / 16;
+
+const top = script => `
+  // place at middle of text line height
+  top: ${halfLineHeightRem(script.doublePica.groupA)}rem;
+
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
+    top: ${halfLineHeightRem(script.doublePica.groupB)}rem;
+  }
+
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    top: ${halfLineHeightRem(script.doublePica.groupD)}rem;
+  }
+
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    top: 0;
+  }
+`;
+
+const SectionDividerWrapper = styled.div`
+  position: relative;
+  margin-top: ${GEL_SPACING_TRPL};
+  margin-bottom: ${GEL_SPACING_DBL};
+  &::before {
+    content: '';
+    position: absolute;
+    height: 1px;
+    border: none;
+    background: ${C_CLOUD_LIGHT};
```

Comment:
In zeplin its using the colour `#aeaeb5`, is this wrong?

Reply:
Caught up with UX, the colour has changed from C_CLOUD_LIGHT to C_PEBBLE (`#aeaeb5`)... except C_PEBBLE doesn't exist in `psammead-styles`, so I raised #501